src/tooltip.js

```diff
@@ -86,6 +86,10 @@ angular.module('cp-tooltip')
 					tooltipDisplayed = false;
 					if(tooltipEl) tooltipEl.remove();
 				}
+
+				function encodeHTML(str) {
+					return str.replace(/&/g, '&amp;').replace(/</g, '&lt;').replace(/"/g, '&quot;');
+				}
```

Comment:
To sanitize html (in a very simple way). The current use case is we're using a tooltip to display some user-inputted data and in theory they could type in something like `<script>alert(1)</script>` or whatever, then when the tooltip renders, the script runs. Or do you think that the sanitization/encoding should happen before the value gets passed to the tooltip to display?